schema/Readme.md

```diff
@@ -86,23 +86,12 @@ your datasources with Lift and administer your data using Studio.
 
 ## Datasource Block
 
-Datasource blocks tell the schema where the models are backed. You can have multiple datasources with different connectors.
+The datasource block tells the schema where the models are backed.
 
 ```groovy
 datasource pg {
   provider = "postgresql"
-  url      = env.POSTGRES_URL
-  enabled  = true
-}
-
-datasource mgo {
-  provider = "mongodb"
-  url      = env.MONGO_URL
-}
-
-datasource mgo2 {
-  provider = "mongodb"
-  url      = env.MONGO2_URL
+  url      = env.DATABASE_URL
```

Comment:
Wasn't this changed to a function?

Reply:
Yes. I just noticed. I'll update the PR accordingly